From c7469ae59e6a33c1550602387fc55dad2570c831 Mon Sep 17 00:00:00 2001
From: Dineshkumar V <dineshkumar.varadarajan@adlinktech.com>
Date: Thu, 5 Dec 2019 18:24:44 +0530
Subject: [PATCH] Added temporary fix to avoid compilation issue for systemd
 recipe

---
 src/basic/missing.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/basic/missing.h b/src/basic/missing.h
index dddc6de..b3e45c9 100644
--- a/src/basic/missing.h
+++ b/src/basic/missing.h
@@ -1405,7 +1405,7 @@ struct fib_rule_uid_range {
 #define PF_KTHREAD 0x00200000
 #endif
 
-#if ! HAVE_STRUCT_STATX
+#if ! HAVE_STRUCT_STATX_IN_SYS_STAT_H
 struct statx_timestamp {
         int64_t tv_sec;
         uint32_t tv_nsec;
-- 
2.7.4

