From e9ab6878def2deb2e621854691468a485d1230fe Mon Sep 17 00:00:00 2001
From: Dineshkumar V <dineshkumar.varadarajan@adlinktech.com>
Date: Thu, 12 Mar 2020 21:15:36 +0530
Subject: [PATCH] Added Adlink wallpaper

---
 backgrounds/Makefile.am | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/backgrounds/Makefile.am b/backgrounds/Makefile.am
index 35857a2..2eee901 100644
--- a/backgrounds/Makefile.am
+++ b/backgrounds/Makefile.am
@@ -4,9 +4,7 @@
 backgroundsdir = $(datadir)/backgrounds/xfce
 
 backgrounds_DATA = \
-    xfce-blue.jpg \
-    xfce-stripes.png \
-    xfce-teal.jpg
+    adlink.jpg
 
 
 EXTRA_DIST = $(backgrounds_DATA)
-- 
2.7.4

